api/scripts/reprocess_transfers.py

```diff
@@ -10,52 +10,57 @@
 
 
 @click.command()
 @click.option('-s', '--start-block-number', 'startBlockNumber', required=True, type=int)
 @click.option('-e', '--end-block-number', 'endBlockNumber', required=True, type=int)
-async def reprocess_transfers(startBlockNumber: int, endBlockNumber: int):
+@click.option('-b', '--batch-size', 'batchSize', required=False, type=int, default=1000)
+async def reprocess_transfers(startBlockNumber: int, endBlockNumber: int, batchSize: int):
     databaseConnectionString = Database.create_psql_connection_string(username=os.environ["DB_USERNAME"], password=os.environ["DB_PASSWORD"], host=os.environ["DB_HOST"], port=os.environ["DB_PORT"], name=os.environ["DB_NAME"])
     database = Database(connectionString=databaseConnectionString)
     saver = Saver(database=database)
     retriever = Retriever(database=database)
     requester = Requester()
+    #NOTE Change to aws credentials before final push
+    workQueue = SqsMessageQueue(region='us-east-1', accessKeyId=os.environ['AWS_KEY'], accessKeySecret=os.environ['AWS_SECRET'], queueUrl='https://sqs.us-east-1.amazonaws.com/113848722427/FemiKiBa')
+    tokenQueue = SqsMessageQueue(region='us-east-1', accessKeyId=os.environ['AWS_KEY'], accessKeySecret=os.environ['AWS_SECRET'], queueUrl='https://sqs.us-east-1.amazonaws.com/113848722427/FemiKiBa')
     slackClient = SlackClient(webhookUrl=os.environ['SLACK_WEBHOOK_URL'], requester=requester, defaultSender='worker', defaultChannel='notd-notifications')
     awsRequester = AwsRequester(accessKeyId=os.environ['AWS_KEY'], accessKeySecret=os.environ['AWS_SECRET'])
     ethClient = RestEthClient(url='https://nd-foldvvlb25awde7kbqfvpgvrrm.ethereum.managedblockchain.eu-west-1.amazonaws.com', requester=awsRequester)
-    blockProcessor = BlockProcessor(ethClient=ethClient)
     notdManager = NotdManager(blockProcessor=None, saver=saver, retriever=retriever, workQueue=None, tokenManager=None, requester=requester, revueApiKey=None)
 
     await database.connect()
-    await slackClient.post(text=f'reprocess_transfers → 🚧 started: {startBlockNumber}-{endBlockNumber}')
-    try:
-        currentBlockNumber = startBlockNumber
-        while currentBlockNumber < endBlockNumber:
-            logging.info(f'Working on {currentBlockNumber}')
-            try:
-                retrievedTokenTransfers = await blockProcessor.process_block(blockNumber=currentBlockNumber)
-            except Exception as exception:
-                logging.info(f'Got exception whilst getting blocks: {str(exception)}. Will retry in 10 secs.')
-                time.sleep(60)
-                currentBlockNumber -= 1
-                continue
-            await notdManager._save_block_transfers(blockNumber=currentBlockNumber, retrievedTokenTransfers=retrievedTokenTransfers)
-            currentBlockNumber = currentBlockNumber + 1
-        await slackClient.post(text=f'reprocess_transfers → ✅ completed : {startBlockNumber}-{endBlockNumber}')
```

Comment:
can you please add these notifications back - so have one at the start just after the .connect()s and then everything in a try catch and if it passes then success and if it fails then error (just copy paste from before)